tools/web3py-example/scripts/test.py

```diff
@@ -0,0 +1,134 @@
+#
+# Hedera JSON RPC Relay - Web3py Example
+#
+# Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import unittest
+from dotenv import load_dotenv
+from web3 import Web3
+from solcx import install_solc, compile_files
+
+def setup_environment():
+    # install latest solc
+    install_solc(version='latest')
+
+    # load values from our .env file
+    load_dotenv()
+    OPERATOR_PRIVATE_KEY = os.getenv('OPERATOR_PRIVATE_KEY')
+    RELAY_ENDPOINT = os.getenv('RELAY_ENDPOINT')
+
+    # connect to chain
+    w3 = Web3(Web3.HTTPProvider(RELAY_ENDPOINT))
+
+    # get account from pk
+    acc = w3.eth.account.from_key(OPERATOR_PRIVATE_KEY)
+
+    return w3, acc
+
+
+def get_balance(w3, acc):
```

Comment:
Could we add some sort of documentation that indicates the argument and return types? Even something informal would be worth it.

Reply:
Done, I hope comments like this are okay